packages/node_modules/@node-red/runtime/lib/nodes/Node.js

```diff
@@ -34,6 +34,9 @@ function Node(n) {
     if (n._alias) {
         this._alias = n._alias;
     }
+    if (n.env) {
```

Comment:
I don't think this is still needed anymore.

Reply:
removed this code.